### PR TITLE
Simplify code in AddCommandParser

### PR DESCRIPTION
**Pull request by Lan-Jingbo**

### CODE REVIEW

File: src/main/java/seedu/guest/logic/parser/AddCommandParser.java [hunk 1]
```diff
@@ -46,29 +46,20 @@ public AddCommand parse(String args) throws ParseException {
...
-            Phone phone = ParserUtil.parsePhone(argMultimap.getValue(PREFIX_PHONE).get());
-            Email email = ParserUtil.parseEmail(argMultimap.getValue(PREFIX_EMAIL).get());
-            Room room = ParserUtil.parseRoom(argMultimap.getValue(PREFIX_ROOM).get());
-            DateRange dateRange = ParserUtil.parseDateRange(argMultimap.getValue(PREFIX_DATE_RANGE).get());
-            NumberOfGuests numberOfGuests = ParserUtil
-                    .parseNumberOfGuests(argMultimap.getValue(PREFIX_NUMBER_OF_GUESTS).get());
-            IsRoomClean isRoomClean = new IsRoomClean(DEFAULT_IS_ROOM_CLEAN);
-            Bill bill = new Bill();
             Request request = ParserUtil.parseRequest(argMultimap.getValue(PREFIX_REQUEST).get());
             Guest guest = new Guest(name, phone, email, room, dateRange, numberOfGuests, isRoomClean, bill, request);
```
Comment (bryanhce) [inline src/main/java/seedu/guest/logic/parser/AddCommandParser.java]: The lines "Guest guest = new Guest(name, phone, email, room, dateRange, numberOfGuests, isRoomClean, bill, request);" and " return new AddCommand(guest);" are duplicated. Should we remove it from the if-else block?

**Review by bryanhce — CHANGES_REQUESTED** [review]
1 change!

File: src/main/java/seedu/guest/logic/parser/AddCommandParser.java (hunk 1 above)
Comment (Lan-Jingbo) [reply src/main/java/seedu/guest/logic/parser/AddCommandParser.java]: cannot, because it will cause error

Comment (bryanhce) [reply src/main/java/seedu/guest/logic/parser/AddCommandParser.java]: Perhaps you need to place the declaration of request outside of the if else block for it to work?

**Review by bryanhce — CHANGES_REQUESTED** [review]

**Review by bryanhce — APPROVED** [review]
LGTM

**Pull request merged**